### PR TITLE
Add test case for rename file

### PR DESCRIPTION
**Pull request by jdneo**
This is to prevent the regression in the future, for the issue: #1517

Signed-off-by: Sheng Chen <sheche@microsoft.com>

### CODE REVIEW

File: test/standard-mode-suite/rename.test.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+	suiteSetup(async function() {
+		getJavaConfiguration().update("refactor.renameFromFileExplorer", "autoApply");
+		await extensions.getExtension('redhat.java').activate();
+	});
+
+	test('rename on file will update the class name', async function () {
+		const workspaceEdit: WorkspaceEdit = new WorkspaceEdit();
+		workspaceEdit.renameFile(Uri.file(originFilePath), Uri.file(newFilePath));
+		await workspace.applyEdit(workspaceEdit);
```
Comment (testforstephen) [inline test/standard-mode-suite/rename.test.ts:24]: The unit test will modify the test project file in the source code directly. Can you copy it to some temporary directory and modify there?

**Review by fbricon — CHANGES_REQUESTED** [review]
Change the test so that it doesn't mutate the sources

File: test/standard-mode-suite/rename.test.ts (hunk 1 above)
Comment (jdneo) [reply test/standard-mode-suite/rename.test.ts:24]: ok

**Comment by jdneo** [comment]
@fbricon Would you mind to tag this PR to a later milestone? I need sometime to figure out if there is some elegant way to prepare the test folder and launch the tests. (Launch from `npm run test` and VS Code launch configuration have different entrances).

**Comment by jdneo** [comment]
Updated

**Comment by fbricon** [comment]
Thanks @jdneo!

**Pull request merged**